features/member_edits_company_page.feature

```diff
@@ -6,17 +6,21 @@ Feature: As a member
 
   Background:
     Given the following users exists
-      | email               | admin | is_member |
-      | emma@happymutts.com |       | true      |
-      | admin@shf.se        | true  | true      |
+      | email                | admin | is_member |
+      | emma@happymutts.com  |       | true      |
+      | alpha@happymutts.com |       | true      |
+      | admin@shf.se         | true  | true      |
 
     And the following companies exist:
       | name                 | company_number | email                  |
       | No More Snarky Barky | 2120000142     | snarky@snarkybarky.com |
 
     And the following applications exist:
-      | first_name | user_email          | company_number | status   | category_name |
-      | Emma       | emma@happymutts.com | 5562252998     | Accepted | Awesome       |
+      | first_name | user_email           | company_number | status   | category_name |
+      | Emma       | emma@happymutts.com  | 2120000142     | Accepted | Awesome       |
+      | Alpha      | alpha@happymutts.com | 5562252998     | Accepted | Awesome       |
+
```

Comment:
Need to changed "Accepted" to "Godkand" (with the appropriate umlaut, of course)